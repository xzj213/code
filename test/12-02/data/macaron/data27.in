18
0 0 0 2801 1453 0 2801 1453 0 6 2801 0 0 2959 0 6 2801 1453
1453 2801 0 0 2801 2801 6 6 6 0 1453 0 0 2801 6 1453 1453 6
2801 6 6 1453 0 0 0 6 0 1453 6 2801 6 6 2801 0 2801 0
1453 6 6 6 0 1453 0 0 0 6 0 0 1453 0 6 6 2801 0
1453 2801 1453 1453 0 1453 0 2801 2801 6 1453 6 0 0 6 1453 2801 1453
2801 2801 2801 2801 0 1453 1453 1453 1453 1453 6 1453 1453 0 6 0 1453 0
1453 6 6 6 6 1453 0 6 1453 1453 0 1453 1453 6 2801 2801 2801 1453
6 1453 6 1453 0 1453 6 6 3498 2801 1453 1453 1453 2801 2801 0 1453 0
6 0 1453 2801 0 2801 0 0 2801 2801 6 0 1453 6 0 0 6 0
6 1453 6 0 2801 6 0 1453 2973 2801 2801 6 1453 2801 0 6 1453 2801
2801 1453 6 2801 6 2801 2801 2801 6 2801 6 1453 1453 1453 1453 0 1453 6
1453 1453 1453 0 1453 1453 0 6 2801 0 6 1453 0 6 2801 2801 0 2801
1453 0 1453 6 2801 1453 2801 2801 0 2801 1453 0 1453 2801 0 1453 2801 0
0 0 1453 0 2801 1453 1453 1453 0 1453 2801 6 2801 6 0 6 6 0
1453 2801 6 6 2801 6 2801 2801 1453 0 2801 1453 1453 2801 1453 2801 1453 1453
2801 2801 6 6 0 2801 2801 0 1453 0 3010 6 6 1453 1453 0 1453 6
6 1453 6 6 2801 1453 2801 6 0 897 6 0 1453 0 1453 0 2801 1453
6 0 2801 6 6 2801 0 0 2801 1453 0 6 1453 0 0 0 0 1453
