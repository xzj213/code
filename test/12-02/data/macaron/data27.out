0 6 9 15 18 20 66 68 75 77 89 95 289 295 307 309 314 316 362 364 369 375 376 382 672 678 690 692 699 701 747 749 752 758 761 767 897 903 904 910 915 917 963 965 970 972 984 990 1162 1164 1169 1175 1176 1182 1217 1223 1235 1237 1242 1244 1442 1444 1451 1453 1465 1471 1504 1510 1513 1519 1522 1524 1571 1573 1578 1580 1592 1598 1633 1639 1640 1646 1651 1653 1803 1805 1808 1814 1817 1823 1856 1862 1874 1876 1883 1885 2049 2055 2056 2062 2074 2076 2115 2117 2122 2124 2129 2135 2345 2351 2354 2356 2363 2365 2402 2404 2416 2422 2425 2431 2728 2734 2739 2741 2746 2748 2787 2789 2801 2807 2808 2814 2944 2950 2953 2959 2971 2973 3010 3012 3019 3021 3024 3030 3202 3204 3216 3222 3225 3231 3273 3279 3282 3284 3291 3293 3491 3493 3498 3500 3505 3511 3553 3559 3560 3566 3578 3580 3618 3620 3627 3629 3632 3638 3680 3686 3689 3695 3707 3709 3843 3845 3857 3863 3864 3870 3912 3918 3923 3925 3930 3932 
